backend/tracim_backend/models/tag.py

```diff
@@ -0,0 +1,60 @@
+from sqlalchemy import Column
+from sqlalchemy import ForeignKey
+from sqlalchemy import Sequence
+from sqlalchemy import Unicode
+from sqlalchemy import UniqueConstraint
+from sqlalchemy.orm import relationship
+from sqlalchemy.types import Integer
+
+from tracim_backend.models.data import Content
+from tracim_backend.models.data import User
+from tracim_backend.models.data import Workspace
+from tracim_backend.models.meta import DeclarativeBase
+from tracim_backend.models.mixins import CreationDateMixin
+
+
+class Tag(CreationDateMixin, DeclarativeBase):
+    __tablename__ = "tag"
```

Comment:
table name are in plural forms for (almost?) all database tables

Reply:
We have both, workspaces and users are plural, but reaction and content are not and I largely imitated reactions here. So, I don't really know what is better. We might want to rename content and reaction at some point for consistency. Should we already go for the plural for tags now?

Reply:
I looked at a few models and (unfortunately) did only find plural-formed tables. I'm pretty sure there is a coding rule for this (I wrote it in early stage of development in some documentation); disorder already there, so ... not required to rename *now*.

Reply:
ok. This is not too costly to do if needed :-)